Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -1445,6 +1442,15 @@ open class ComposerVC: _ViewController,
         guard textView.text != content.text else { return }
 
         content.text = textView.text
+
+        guard let ephemeralMessageId = self.ephemeralMessageId else {
+            channelController?.createEphemeralMessage(text: content.text) { [weak self] result in
+                self?.ephemeralMessageId = try? result.get()
+            }
+            return
+        }
+
+        channelController?.updateEphemeralMessage(id: ephemeralMessageId, text: content.text)
```

Comment:
When updating the text, we need to check if the ephemeral message is already created. If not, we create it, if yes, we update it.

Hopefully we can improve this, and handle this internally.

Reply:
New WIP simpler version, does not require all this logic.